src/components/EditEvent/EventInformation.tsx

```
@@ -1,27 +1,49 @@
-import DatePicker from "wowds-ui/SingleDatePicker";
-import TextField from "wowds-ui/TextField";
-import TimePicker from "wowds-ui/TimePicker";
 import { Flex } from "@/components/@common/Flex";
 import { Space } from "@/components/@common/Space";
 import { Text } from "@/components/@common/Text";
 import { EventType } from "@/types/dtos/event";
 
-const parseISO = (s?: string): Date | undefined => (s ? new Date(s) : undefined);
-const toISOOrEmpty = (d?: Date) => (d ? d.toISOString() : "");
+const parseISO = (s?: string): Date | undefined => {
+  if (!s) return undefined;
```

ESLint

[error] Expected { after 'if' condition.

(curly)